graph_weather/models/fengwu_ghr/layers.py

```diff
@@ -139,8 +140,20 @@ def __init__(self, *, image_size,
                  patch_size, depth, heads,
                  mlp_dim, channels, dim_head,
                  res=False,
-                 scale_factor=None):
+                 scale_factor=None,
+                 **kwargs):
         super().__init__()
+        #TODO this can probably be done better
```

Comment:
I think you can do this with `dataclass` on the object, pretty sure it autodoes this for you then